### PR TITLE
fix for site permissions menu dropdown - remove z-index css in panel.css and callout.css 

### PR DESCRIPTION
**Pull request by abenems**

### CODE REVIEW

File: src/components/Persona/Persona.tsx [hunk 1]
```diff
@@ -23,19 +23,29 @@ export class Persona extends React.Component<IPersonaProps, any> {
...
+      primaryText,
+      secondaryText,
+      tertiaryText,
+      optionalText,
+      hidePersonaDetails
+    } = this.props;
 
     return (
-      <div { ... this.props as any } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
+      <div { ...this.props as any } className={ css('ms-Persona', className, PERSONA_SIZE[size], PERSONA_PRESENCE[presence]) }>
```
Comment (dzearing) [inline src/components/Persona/Persona.tsx]: I'm confused. Have you merged latest changes? I thought i had fixed this:

Add this import:

```typescript
import { getNativeProps, divProperties } from '../../Utilities';
```

Then avoid mixing in this.props directly. Instead mix in DIV-safe props only:

```typescript
    let divProps = getNativeProps(this.props, divProperties);

    return <div { ...divProps } ... />
```

Comment (abenems) [inline src/components/Persona/Persona.tsx]: yes, sorry, I just saw it :)

Comment (abenems) [inline src/components/Persona/Persona.tsx]: sorry David, I shouldn't make changes to the Persona.tsx ...sorry, I picked up my old changes, I should only have changes for the Panel.css and Callout.css

Comment (abenems) [inline src/components/Persona/Persona.tsx]: updated the PR to reflect only the changes in Panel.scss and Callout.scss

**Comment by dzearing** [comment]
Approved

**Pull request merged**